app/concepts/todo/create.rb

```diff
@@ -1,50 +1,38 @@
 class Todo::Create < Trailblazer::Operation
+  include Model
+  model Todo, :create
+
+  contract do
+    property :title, validates: { presence: true }
+  end
 
-  # Logic for creating a TODO
-  #     if `todo_list` is specified
-  #         create the `todo`
-  #         associated with `todo_list`
-  #     else
-  #         create the `todo`
-  #         create a `user`
-  #         create a default `todo_list`
-  #         associate `todo_list` with `user`
-  #         associate `todo` with `todo_list`
   def process(params)
-    # make sure title is not blank
-    return invalid! if params[:todo][:title].blank?
+    validate(params[:todo]) do
+      contract.model.list = get_or_create_todo_list(params)
+      contract.model.list.user = get_or_create_user(params)
+      contract.model.list.save
+      contract.save
+    end
+  end
```

Comment:
I'm thinking to move those 3 lines to a method `update_todo_list`, something like this:
```
  def update_todo_list(todo, params)
    todo.list = get_or_create_todo_list(params)
    todo.list.user = get_or_create_user(params)
    todo.list.save
  end
```
I'm thinking if it can be something like todo.update_list(params), but in Trailblazer, the model `Todo` is only concerned with persisting data and shouldn't contain these logic right?